cjworkbench/i18n/trans.py

```diff
@@ -248,6 +214,27 @@ def for_application(self) -> MessageLocalizer:
         """Return a `MessageLocalizer` for the application messages"""
         return self._app_localizer
 
+    def _create_localizer_for_module_zipfile(
+        cls, module_zipfile: ModuleZipfile
+    ) -> MessageLocalizer:
+        catalogs = {}
+        for locale_id in supported_locales:
+            try:
+                catalogs[locale_id] = read_po(
+                    BytesIO(module_zipfile.read_messages_po_for_locale(locale_id)),
+                    abort_invalid=True,
+                )
+            except PoFileError as err:
+                logger.exception(
+                    f"Invalid po file for module {module_zipfile.module_id_and_version} in locale {locale_id}: {err}"
+                )
+                catalogs[locale_id] = Catalog()
+            except KeyError as err:
+                pass
+        if not catalogs:
+            raise NotInternationalizedError(module_zipfile.module_id)
+        return MessageLocalizer(MessageCatalogsRegistry(catalogs))
+
```

Comment:
Can we remove `MessageCatalogsRegistry`? It's _only_ used in `MessageLocalizer`, and as far as I can tell it's one method and a dict.

Reply:
So I move the dict and the method back to `MessageLocalizer`?